rspace/src/main/scala/coop/rchain/rspace/nextgenrspace/RSpaceOps.scala

```diff
@@ -30,6 +31,8 @@ abstract class RSpaceOps[F[_]: Concurrent, C, P, A, R, K](
     logF: Log[F]
 ) extends SpaceMatcher[F, C, P, A, R, K] {
 
+  protected[this] val eventLog: SyncVar[EventLog] = create[EventLog](Seq.empty)
```

Comment:
🎉

Reply:
can we avoid the syncvar, and hava a Ref/MVar?

Reply:
Not in this PR. Here it is merely moved from a less appropriate place

Reply:
I think we have this ticketed